src/component/Flow.test.ts

```diff
@@ -22,12 +22,18 @@ import Plumber from '../services/Plumber';
 import { ConnectionEvent, createStore, initialState } from '../store';
 import { getCollisions, getGhostNode, getRenderNodeMap } from '../store/helpers';
 import { createSetup, createSpy, getSpecWrapper } from '../testUtils';
-import { getBaseLanguage } from '../utils';
+import { getBaseLanguage, dump } from '../utils';
 
 jest.mock('../services/ActivityManager');
 jest.mock('../services/Plumber');
 jest.useFakeTimers();
 
+jest.mock('uuid', () => {
```

Comment:
this looks like learning opportunity, why might we want to hijack `v4`/`generateUUID`?

Reply:
in part so that we can generate snaps?

Reply:
this is just for deterministic tests, and ya, snaps especially

Reply:
should we add this to `__mocks__`? or do you see this as something we'd want to do in select cases?

Reply:
I imagine tests will want to reset the counter at various points. Is there a way to get it to reset itself without putting it in `beforeEach`?

Reply:
we can reset it between each test using the [clearMocks](https://facebook.github.io/jest/docs/en/configuration.html#clearmocks-boolean) setting. can verify that works w/ our axios mock in `__mocks__`.